salt/modules/virt.py

```diff
@@ -3880,44 +3847,66 @@ def define_vol_xml_path(path, pool=None, **kwargs):
 
 def migrate_non_shared(vm_, target, ssh=False, **kwargs):
     """
-    Attempt to execute non-shared storage "all" migration
+    .. deprecated:: 3002 Use ``virt.migrate`` with ``copy_storage='all'`` instead.
```

Comment:
Use the codename (Magnesium IIRC) instead of the number

Reply:
This is something I think should be improved. We used to use the codename because we were using date based versions so we were never certain what the version would be in the future. Now we know what the version will be in the future, so in my opinion it would be better to use the version number going forward. All we do at release time is search for the codename references and change them to the correct version number. I'll propose this in a PR with an update to the docs. But for this PR its fine to use either but i'll add a PR to try to improve things in the future.